sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -63,7 +64,9 @@
 @ServiceClientBuilder(serviceClients = CryptographyClient.class)
 public final class CryptographyClientBuilder {
     final List<HttpPipelinePolicy> policies;
+    final UserAgentProperties properties;
     private final ClientLogger logger = new ClientLogger(CryptographyClientBuilder.class);
+    private static final String AZURE_KEY_VAULT_KEYS = "azure-key-vault-keys.properties";
```

Comment:
Does these file name need to be documented in README or documentation so user know about these names ?

Reply:
The version is internal use stuff. README is for public to use. I don't think it needs to put in README. What do you think? Or I can leave a comment above the line.